docs/authoring-recipes/multiple-versions.md

```diff
@@ -44,22 +44,18 @@ The benefit of a TypeTable is that you don't need to pull down the entire JAR to
 
 #### Using TypeTables in your tests
 
-With the TypeTable file created, you can use the `classpathFromResources` function in your tests. For instance, if you want to default to using `junit-4.13.2` and `mockito-core-3.12.4` in your tests, you could add this to your test file:
+With the TypeTable file created, you can use the `classpath(String...)` function in your tests. For instance, if you want to default to using `junit-4.13.2` and `mockito-core-3.12.4` in your tests, you could add this to your test file:
```

Comment:
I feel there's some potential for confusion here: folks do not need type tables to use `classpath(String...)`; perhaps best to explain type tables only in the context of using JavaTemplate and JavaParser in a recipe, as that's what it's intended for, and remove the references to testing here.

Reply:
👍 alright removed this section.